package.json

```
@@ -60,7 +61,7 @@
     "typescript": "^4.8.4"
   },
   "peerDependencies": {
-    "@capacitor/core": "^3.0.0 || ^4.0.0"
+    "@capacitor/core": "5.0.0-beta.2" 
```

```suggestion
    "@capacitor/core": "^5.0.0" 
```

Should be possible now with the release.